engine/src/main/java/org/terasology/rendering/world/ChunkMeshUpdateManager.java

```diff
@@ -168,17 +166,14 @@ public boolean isTerminateSignal() {
 
         @Override
         public void run() {
-            ChunkMesh[] newMeshes = new ChunkMesh[numberOfMeshSegments];
+            ChunkMesh newMesh;
```

Comment:
Note to self: this had been moved out of a for loop, but as the for loop has been removed, it can now go back in the if block.